vocabularies/sample-type.ttl

```diff
@@ -138,6 +144,20 @@ sampty:core a skos:Concept ;
     skos:prefLabel "Core"@en ;
     skos:topConceptOf <http://linked.data.gov.au/def/sample-type> .
 
+sampty:hcore a skos:Concept ;
+    skos:definition "A core sample that has been cut in half parallel to its length."@en ;
+    skos:inScheme <http://linked.data.gov.au/def/sample-type> ;
+    skos:notation "HCORE" ;
+    skos:prefLabel "Half Core"@en ;
+    skos:broader sampty:core .
+
+sampty:qcore a skos:Concept ;
+    skos:definition "A core sample that has been cut in quarters parallel to its length."@en ;
+    skos:inScheme <http://linked.data.gov.au/def/sample-type> ;
+    skos:notation "QCORE" ;
+    skos:prefLabel "Quarter Core"@en ;
+    skos:broader sampty:core .
+

```

Comment:
The letters H and P are used in combination with other letters to define core barrel diameter and therefore drillcore diameters. Will this be confusing? 

See https://www.drillingsupplystore.com/coring-in-hole-tools.html

Reply:
It shouldn't be an issue.  These notations are just for defining our URIs for each sample. The barrel diameter will be recorded separately.

Reply:
All good. Tech good.